tests/resources/openassetio_config_cpp.toml

```diff
@@ -0,0 +1,18 @@
+# This config sets up the SimpleCppManager C++ manager/plugin for use in
+# the main test cases.
+
+[manager]
+identifier = "org.openassetio.examples.manager.simplecppmanager"
+
+[manager.settings]
+# Pretend to be BasicAssetLibrary aka BAL. This is so a single set of
+# USD files can be used for both Python and C++ testing.
+prefix = "bal:///"
+
+read_traits = '''
+bal:///not_a_file,openassetio-mediacreation:content.LocatableContent,location,https://stuffonmycat.com
```

Comment:
https://stuffonmycat.com ??

realise this is a test, but everything else in the file looks like it _could_ be in a production asset.

Reply:
:laughing: it matches the corresponding entry in `bal_library.json` - blame Tom's whimsy